Prepares watchmaker v0.21.5 release (#1555)
plus3it/watchmaker: CHANGELOG.md

## Changelog

### 0.21.5

**Commit Delta**: [Change from 0.21.4 release](https://github.com/plus3it/watchmaker/compare/0.21.4...0.21.5)

**Released**: 2021.02.25

**Summary**:

*   ash-linux-formula
    - Replace `watch` with `listen` to restart the `sshd` service a single time
    - Make state RHEL-07-040560 more resilient when the yum group info is missing
*   scap-formula
    - Updates SCAP content from DISA (as of February 2021) and OpenSCAP (v0.1.54)
*   Update watchmaker default `config.yaml` to use salt v2019.2.8

> 

Reviewer: Add mention of ability to browse Cloudarmor repo

```suggestion
*    Ability to browse [Watchmaker Cloudarmor repo](https://watchmaker.cloudarmor.io/list.html)
```

Reviewer: @kumoy There are some merge stragglers.

## Changelog

### 0.21.5

**Commit Delta**: [Change from 0.21.4 release](https://github.com/plus3it/watchmaker/compare/0.21.4...0.21.5)

**Released**: 2021.02.25

**Summary**:

*   ash-linux-formula

>     - Replace watch with listen to address service restarts

Reviewer 2: ```suggestion
    - Replace `watch` requisite with `listen` to restart the `sshd` service a single time
```

    - Make state RHEL-07-040560 more resilient when the yum group info is missing
*   scap-formula
    - Updates SCAP content from DISA (as of February 2021) and OpenSCAP (v0.1.54)

> *   watchmaker-salt-content

Reviewer 2: We didn't actually modify the `watchmaker-salt-content` submodule. That's a separate repo. We modified the default `config.yaml`, which is part of the watchmaker repo.

```suggestion
*   Update watchmaker default `config.yaml` to use salt v2019.2.8
```

    - Bump salt version to 2019.2.8

> *   Bump readme-renderer from 28.0 to 29.0

Reviewer 2: We can remove this. We don't generally list out changes that are not user-facing. This one is only relevant to the CI/packaging processes...
